gallery/2021/candid/candid.pug

```diff
@@ -0,0 +1,150 @@
+include mixins/active-link
```

Comment:
If you use `extends /base`, you won't have to rewrite the same code over and over.

Plus, it keeps the pug files more modular that way so we don't have to, for example, edit the navbar in every single pug file.